src/commands/Music/join.ts

```diff
@@ -35,27 +36,27 @@ export default class extends MusicCommand {
 			// Connect to Lavalink and join the voice channel
 			await audio.connect(channel.id);
 		} catch {
-			return message.sendLocale(LanguageKeys.Commands.Music.JoinFailed);
+			return message.sendTranslated(LanguageKeys.Commands.Music.JoinFailed);
 		}
 
-		return message.sendLocale(LanguageKeys.Commands.Music.JoinSuccess, [{ channel: `<#${channel.id}>` }]);
+		return message.sendTranslated(LanguageKeys.Commands.Music.JoinSuccess, [{ channel: `<#${channel.id}>` }]);
```

Comment:
In case Discord implements a way to mention channels, it'll be better than plain text, and we won't have to change a thing.

Also, unlike sending the name, the channel's name updates, even if it doesn't mention.